bin/pt-archiver

```diff
@@ -7179,14 +7292,14 @@ sub main {
             my $lag_dbh = $lag_server->{'dbh'};
             my $id      = $lag_server->{'id'};
             if ( $lag_dbh ) {
-               my $lag = $ms->get_slave_lag($lag_dbh);
+               my $lag = $ms->get_replica_lag($lag_dbh);
                while ( !defined $lag || $lag > $o->get('max-lag') ) {
                   PTDEBUG && _d("Sleeping: slave lag for server '$id' is", $lag);
```

Comment:
Since pt-heartbeat had his schema changed to reflect the source/replica change, does this one should be changed for coherence ?
Or, is it left as is so to keep the current output ?

Reply:
Hi! I don't understand your comment. Modified code uses get_replica_lag. What needs to be changed?

Reply:
It is for `_d("Sleeping: slave lag for server '$id' is", $lag);`, on row 7297 and 7299 (on new branch).
7297 is with PTDEBUG, so really not a problem, but the other one is a log to users
"replica lag" would make it more coherent, but that's only a nitpick

Reply:
Fixed, please review again

Reply:
lgtm for the bin, the tests will need some fixing as well
```
$ grep -rn 'Sleeping:'
t/pt-archiver/replica_lag.t:73:   qr/Sleeping: slave lag for server/,
t/pt-archiver/replica_lag.t:101:   qr/Sleeping: slave lag for server/,
t/pt-archiver/replica_lag.t:129:   qr/Sleeping: slave lag for server/,
t/pt-archiver/replica_lag.t:156:   qr/Sleeping: slave lag for server/,
t/pt-archiver/replica_lag.t:184:   qr/Sleeping: slave lag for server/,
t/pt-archiver/replica_lag.t:212:   qr/Sleeping: slave lag for server/,
t/pt-archiver/replica_lag.t:240:   qr/Sleeping: slave lag for server/,
bin/pt-archiver:7297:                  PTDEBUG && _d("Sleeping: replica lag for server '$id' is", $lag);
bin/pt-archiver:7299:                     _d("Sleeping: replica lag for server '$id' is", $lag);
```

Reply:
Fixed